### PR TITLE
Payout actions

### PR DESCRIPTION
**Pull request by vanruch**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] index.js exports updated
- [x] Automatic tests added/updated
- [x] Automatic tests pass
- [x] Sanity check passed
- [x] Self-review passed
- [x] Package version updated

### CODE REVIEW

File: src/tasks/payouts.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    try {
+      if (await await this.payoutsActions.getTotalAvailablePayout() === '0') {
+        console.log('Nothing to withdraw :(');
+        return;
+      }
+      const balanceBefore = this.web3.utils.toBN(await this.web3.eth.getBalance(this.sender));
+      await this.payoutsActions.withdraw();
+      const balanceAfter = this.web3.utils.toBN(await this.web3.eth.getBalance(this.sender));
+      const balanceDiff = this.web3.utils.fromWei(balanceAfter.sub(balanceBefore), 'ether');
+      console.log(`${balanceDiff} have been sent to your account`);
```
Comment (spherefoundry) [inline src/tasks/payouts.js]: This is not strictly true as some funds will be spend on gas. I would rather take the value from `getTotalAvailablePayout` (we already call it) and output that.
